### ISSUE TITLE
[v1.0-Elements-Input.Toggle.json]: Voice over is announcing unnecessary information for 'I accept the terms and agreements' switch button.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8263]
### Target Platforms

iOS

### SDK Version

1.0 (2.4.0-beta.23.1.8.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 16.2
Application: Adaptive Cards
App Version: 1.0 (2.4.0-beta.23.1.8.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.0" control and select it.
5. Navigate to 'Elements' control and activate it.
6. Navigate to 'Input.Toggle.json' under 'Elements' list item and activate it.
7.  Cards will appear.
8. Navigate to 'I accept the terms and agreements' switch button using swipe gesture.
9. Observe the issue.
 
**Actual Result:**
Voice over is announcing unnecessary information for 'I accept the terms and agreements' switch button.

Voice over is announcing as 'left bracket null right bracket I accept the terms and agreements comma switch button off'

**Expected Result: **
Voice over should necessary information for 'I accept the terms and agreements' switch button.

Voice over should announce necessary information as 'I accept the terms and agreements switch button off

**Note:**
This issue is repro throughout application.

**User Experience:**
If content is read unnecessary information to assistive technology, it could interfere with the comprehension of that content or simply make the application frustrating to use.  

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7B9C514377-76E5-4CFE-8300-DA3079D3CDC3%7D&file=MAS%201.3.1%20%E2%80%93%20Info%20and%20Relationships.docx&action=default&mobileredirect=true&share=IQF3Q1Gc5Xb-TIMA2jB5083DAcWOZyaXcx7TtNeRlXM4uHk

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/95913748/215438109-912b19e6-8f9e-4e6e-8102-53a19aea544c.mov

### Card JSON

```JSON
NA
```
